docs/docs/consumer-development/changeover-procedure.md

```diff
@@ -0,0 +1,231 @@
+---
+sidebar_position: 5
+---
+
+# Changeover Procedure
+
+Chains that were not initially launched as consumers of replicated security can still participate in the protocol and leverage the economic security of the provider chain. The process where a sovereign chain transitions to being a replicated consumer chain is called the **changeover procedure** and is part of the interchain security protocol. After the changeover, the new consumer chain will retain all existing state, including the IBC clients, connections and channels already established by the chain.
```

Comment:
Seems "sovereign to consumer" has stuck, but we've committed to something else in the code. See https://github.com/cosmos/interchain-security/pull/757#discussion_r1139884029

Reply:
Good catch!